src/quo2/components/inputs/locked_input/view.cljs

```diff
@@ -0,0 +1,30 @@
+(ns quo2.components.inputs.locked-input.view
+  (:require [react-native.core :as rn]
+            [status-im2.contexts.quo-preview.tabs.segmented-tab]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.components.inputs.locked-input.style :as style]
+            [quo2.theme :as theme]
+            [quo2.components.markdown.text :as text]))
+
+(defn- info-box
+  [theme icon label-text]
+  [rn/view
+   {:style (style/info-box-container theme)}
+   [rn/view {:margin-right 1}
+    [icons/icon icon {:width  20
+                      :height 20
+                      :color  (colors/theme-colors colors/neutral-50
+                                                   colors/neutral-10 theme)}]]
+   [text/text
+    {:style (style/info-box-label theme)} label-text]])
+
+(defn- locked-input-internal
+  [{:keys [label-text label-style value-text icon style theme]}]
```

Comment:
hmm, `label-style` should not be a prop here. This should be encapsulated by the component.
also we have other components where `value` is the 2nd param so you can do something like
`(defn- locked-input-internal
  [{:keys [label-text label-style icon style theme]} value]`
  
 I'm not sure what you are using `style` prop here but most styles should be set by the component. I would also call this `container-style` as it is slightly more accurate. Generally we use this as a convenient way to set margins so we don't have to wrap a component in an `rn/view` when using but anything beyond that should really be internal to the component 👍